src/ObjectUtils/CoffFile.cpp

```diff
@@ -93,7 +93,10 @@ ErrorMessageOr<SymbolInfo> CoffFileImpl::CreateSymbolInfo(
   symbol_info.set_name(name);
   symbol_info.set_demangled_name(llvm::demangle(name));
   symbol_info.set_address(symbol_ref.getValue() + section_offset.value());
-  symbol_info.set_size(symbol_ref.getCommonSize());
```

Comment:
link a bug in the commit message?

Reply:
Oh yes, good point. Of course there is already a bug - I completely missed that. Done.